templates/registration/registration_form.html

```diff
@@ -79,8 +79,9 @@
         #edit-form {
             border: unset;
             background: unset;
-            max-width: 450px;
+            max-width: 100%;
```

Comment:
Does `max-width: 450px` and `width:100%` (for both elements) without the margin hack not work?

Reply:
Yep it doesn't work. The form inputs still go outside the screen.

Reply:
Hmm it seems the form originally was not centred anyways. I would suggest setting `max-width: 450px` and `width: 100%` on `#center-float` and removing both `max-width` and `width` on `#edit-form`, which seems to centre things and doesn't overflow on a small screen without needing the negative margin. Maybe someone else should check to make sure this is reasonable.

Reply:
Removing both `max-width` and `width` on `#edit-form` works. Thanks. I committed my changes.